manager/base.py

```diff
@@ -468,10 +470,13 @@ def parse_str_or_list(str_list):
 
 def get_or_create_account():
     """In some cases rh_account might not have already been created by listener, create it for usage in manager"""
-    rh_account = RHAccount.select(RHAccount.id).where(RHAccount.account_number == connexion.context['user'])
+    rh_account = RHAccount.select(RHAccount.id).where(RHAccount.org_id == connexion.context['user']['org_id'])
```

Comment:
Could we add here `.first()` , so in the child function calling this does not need to do the zero indexing? For example: `status_handler.py` line 111. It is not a bug, but since you are changing these functions :)

Reply:
I moved the zero index .id to this function, is it enough? :smile: